src/js/components/EditsNav.jsx

```diff
@@ -172,6 +19,55 @@ export default class EditsNav extends Component {
       headerHeight: 0,
       editsNavHeight: 0
     }
+    this.navMap = {
+      upload: {
+        isReachable: () => true,
+        isErrored: () => this.props.code === PARSED_WITH_ERRORS,
+        isCompleted: () => this.props.code > VALIDATING,
+        errorClass: 'error',
+        errorText: 'uploaded with formatting errors',
+        completedText: 'uploaded',
+        link: 'upload'
+      },
+      'syntactical & validity edits': {
+        isReachable: () =>
+          this.props.fetched && this.navMap.upload.isCompleted(),
+        isErrored: () => this.props.syntacticalValidityEditsExist,
+        isCompleted: () => !this.props.syntacticalValidityEditsExist,
+        errorClass: 'error',
+        errorText: 'syntactical & validity edits found',
+        completedText: 'no syntactical & validity edits',
+        link: 'syntacticalvalidity'
+      },
+      'quality edits': {
+        isReachable: () =>
+          this.props.fetched &&
+          this.navMap['syntactical & validity edits'].isCompleted(),
```

Comment:
This is causing this link to render when there are parsing errors. The [`isCompleted`](https://github.com/cfpb/hmda-platform-ui/pull/916/files#diff-07bda16239ad4cd288276a99e9a840c4R36) for `syntactical & validity edits` is `true` in this case because there are no syntactical or validity edits.

We'll probably need a `code` check as well.

![parsing-nav](https://user-images.githubusercontent.com/2932572/33396797-a86f13ea-d517-11e7-9e8c-81eb829cd100.png)